translate/rus.json

```diff
@@ -191,15 +191,15 @@
 "highlight size":"размер подсветки",
 "history lag":"отставание в истории",
 "import settings":"импортировать настройки",
-"in progress":"в ходе выполнения",
-"info options":"настройки инфор",
+"in progress":"партия идёт",
+"info options":"настройки инфо",
 "info":"инфо",
 "information":"информация",
 "inverted":"обратный",
 "join next":"соединить",
 "key repeat initial":"начальное повторение клавиши",
 "key repeat":"повторение клавиши",
-"kibitz":"советчик",
+"kibitz(-ers)":"советчик(-и)",
```

Comment:
Usually we have more than one kibitzer, right? This change for both locals addresses this issue. Unfortunately I don't know any other languages, so you have to adjust other translations yourself. Sorry :(